src/WordPress/LoadSandbox.php

```diff
@@ -123,6 +123,12 @@ class_exists(InstallationException::class);
             }
         }
 
+        if ($bodyContent === 'COMMAND DID NOT FINISH PROPERLY.') {
+            // We got here from \Codeception\Subscriber\ErrorHandler::shutdownHandler().
+            // We'll try to provide some clues to the user by adding some debug output.
```

Comment:
```suggestion
            // We'll try to provide some clues to the user in the exception message.
            codecept_debug('Codeception error: ' . $bodyContent . ' Check logs for details.');
```
This would avoid duplication with the exception message but still alert them during `--debug`

Reply:
If we're here, we know that body content is `COMMAND DID NOT FINISH PROPERLY.`; I do not see a lot of value in showing it, but not a lot of harm either.